python/dlr/tf_model.py

```diff
@@ -52,6 +52,9 @@ def _get_input_and_output_names(graph):
     for op in graph.get_operations():
         if not op.name.startswith(op_prefix):
             continue
+        # Ignore operators which are not on the top level (not directly under <PREFIX> node)
+        if op.name.find('/', len(op_prefix)) > -1:
```

Comment:
merge the two continue together? 

`if not op.name.startswith(op_prefix) or op.name.find('/', len(op_prefix)) > -1:`

Reply:
fixed